css/site.css

```diff
@@ -1127,6 +1113,34 @@ hr {
     /*body { font-size: 1.125em; }*/
 }
 
+/* Fancy OL styling for the Source Code Policy */
+li ol {
+    counter-reset: list;
+}
+ol > li {
+    list-style: decimal;
+    text-transform: uppercase;
+}
+ol ol > li {
+    list-style: none;
+    position: relative;
+    text-transform: none;
+}
+ol ol > li:before {
+    counter-increment: list;
+    content: counter(list, lower-alpha) ") ";
+    position: absolute;
+    left: -1.4em;
+}
+ol ol ol > li:before {
+    content: none;
+}
+ol ol ol > li {
+    list-style-type: lower-alpha;
+}
+ol ol ol ol > li {
+    list-style-type: lower-roman;
+}
```

Comment:
Why are you pulling all this `ol` styling in here? It'll mess up simple `ol`s in our articles.